yamux/src/stream.rs

```diff
@@ -391,9 +385,7 @@ impl AsyncWrite for StreamHandle {
         buf: &[u8],
     ) -> Poll<io::Result<usize>> {
         match self.state {
-            StreamState::RemoteClosing | StreamState::Reset => {
```

Comment:
I think there is a problem here, when the remote a is closed, b can only read the remote data and can no longer send data to the remote